src/main/resources/assets/onebar/lang/zh_cn.json

```diff
@@ -0,0 +1,131 @@
+{
+  "modmenu.descriptionTranslation.onebar": "只在你需要时显示长条和数值的界面模组",
+  "text.onebar.health": "命%s",
+  "text.onebar.absorption": "吸%s",
+  "text.onebar.hunger": "饿%s",
+  "text.onebar.resistance": "抗%s%%",
+  "text.onebar.invisibility": "隐",
+  "text.onebar.air": "气%s",
+  "text.onebar.freeze": "冷%s",
+  "text.onebar.fire": "火%s×",
+  "text.onebar.warden": "监%s",
+  "text.onebar.hardcore": "HC",
+  "text.onebar.badOmen": "咒%sx",
+  "text.onebar.mobHeadSkeleton": "骷",
```

Comment:
This should represent "skeleton head", abbreviated in English as "sH". Not sure how abbreviations work in Chinese but maybe you can represent it in two characters somehow?
Similar case for other strings below

Reply:
Compared to "sH" in English, it seems too wide if I use two characters in Chinese like "骷头" or "骷髅" (full string is "骷髅头颅") I thought. So, I use one instead of two to make it as short as English.

Reply:
In here it appears wider yes, but not sure how much it differs in game, I think all letters are the same width in default font for ASCII and therefore would appear similar to Chinese ones... 

I've approved the workflow so you can soon test how the hardcore indicator looks in-game by downloading the latest zip from [Actions](https://github.com/Madis0/OneBar/actions).

Reply:
Okay, it's not quite the same width as I thought, but using two Chinese characters is still fine, it looks about the same as one character + emoji.
![image](https://user-images.githubusercontent.com/8611110/203111823-0e8a2bc8-ca9f-49ac-bd35-4c6377cc2504.png)

Reply:
```suggestion
  "text.onebar.mobHeadSkeleton": "骷髅",
```